demo.py

```diff
@@ -417,26 +192,30 @@ def main():
         with torch.no_grad():
             # compute output heatmap
             output = model(input)
-            coords, maxvals = get_max_preds(output.clone().cpu().numpy())
+            coords, maxvals = m.get_max_preds(output.clone().cpu().numpy())
```

Comment:
I would set demo from webcam by default (and wihtout extra cmd key), and demo on image by cmd key.